dev-util/shc/shc-3.9.6.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A generic shell script compiler"
+DESCRIPTION="Shc takes a script, which is specified on the command line and \
+produces C source code. The generated source code is then compiled and linked \
+to produce a stripped binary executable. \
+The compiled binary will still be dependent on the shell specified in the \
+first line of the shell code (i.e shebang) (i.e. #!/bin/sh), thus shc does not \
+create completely independent binaries."
+HOMEPAGE="https://github.com/neurobin/shc"
+COPYRIGHT="1997-2013 Francisco Garcia
+ 	2015-2017 Md Jahidul Hamid"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/neurobin/shc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="da6a2a3ff4c356a61e086c616561bf681489993cab00c426bad0cfd703a68063"
+SOURCE_FILENAME="shc-$portVersion.tar.gz"
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	shc = $portVersion
+	cmd:shc
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	"
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
+
+TEST()
+{
+	#the produced binary doesn't work when running the test
+	#with haikuporter, running it from Terminal works well
```

Comment:
The runtimes produced by shc need to be able to run the shell interpreter, so I guess that adding `cmd:sh` or `cmd:bash` to `BUILD_REQUIRES` will make the test succeed.
(If a test succeeds outside of the chroot, then it should also work in the chroot, as long as you add the required runtime dependencies.)

Reply:
Running the test isn't the issue here, that part is ok, the produces binary doesn't work :)

  ~/haikuports/dev-util/shc/work-3.9.6/sources/shc-3.9.6/test> ./test
  ./test:UaÐ¶1dóš‰
  ~/haikuports/dev-util/shc/work-3.9.6/sources/shc-3.9.6/test> ./test
  ./test: L.
  F
  ޘ_

Reply:
In haikuports with and without cmd: sh (and bash) ...

Reply:
In Terminal (outside of haikuporter) ... 
  ~/haikuports/dev-util/shc/work-3.9.6/sources/shc-3.9.6/test> shc -v -f test.bash -o test
  shc shll=bash
  shc [-i]=-c
  shc [-x]=exec '%s' "$@"
  shc [-l]=
  shc opts=
  shc: cc  test.bash.x.c -o test
  shc: strip test
  shc: chmod ug=rwx,o=rx test
  ~/haikuports/dev-util/shc/work-3.9.6/sources/shc-3.9.6/test> ./test
  $@ is 
  command line: ./test 
  hello world
  [15288] PAUSED... Hit return!

Reply:
It worked fine for me, on x86_gcc2.
(I'll test on x86_64.)

Reply:
`hp --test shc` also works fine for me on x86_64:
```
----- Package Info ----------------
header size:                     80
heap size:                      765
TOC size:                        41
package attributes size:        851
total size:                     845
-----------------------------------
Testing ...
shc shll=bash
shc [-i]=-c
shc [-x]=exec '%s' "$@"
shc [-l]=
shc opts=
shc: cc  test.bash.x.c -o test
shc: strip test
shc: chmod ug=rwx,o=rx test
$@ is Testing shc-3.9.6 on x86_64.
command line: ./test Testing shc-3.9.6 on x86_64.
hello world
[2928] PAUSED... Hit return!
/data/haikuports/dev-util/shc> echo $?
0
```

Reply:
As explained earlier :) running the test is ok :) (the produced binary is not)